spec/ci_helper.rb

```diff
@@ -16,17 +16,14 @@ def self.start
 
     # Make sure directly in the postgresql.conf, the data_directory is set
     # (requirement for pg_wrapper I think...)
-    system("echo \"data_directory = '/var/ramfs/postgresql/9.5/main'\" >> #{PGCONFIGDIR}/postgresql.conf")
-
-    # Clear out the existing ramfs dir so it will be re-generated on boot
-    system("sudo rm -rf /var/ramfs/postgresql/9.5")
+    system("echo \"data_directory = '/var/ramfs/postgresql/10/main'\" >> #{PGCONFIGDIR}/postgresql.conf")
```

Comment:
This is my fault, but I am wondering why I didn't use `PGDATADIR` here...

Want to change it and pretend that I did it correctly the first time?

Reply:
Well the first time around they weren't the same directory so definitely not your fault. I'll DRY it up.